pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Artificial Intelligence",
 ]
 dependencies = [
-   "interegular",
+   "interegular @ git+https://github.com/MegaIng/interegular.git@113d8a48af01f2659941fa165dbdad53bf4f4f11",
```

Comment:
We can't merge a change like this for compatibility reasons, and because it will limit users from future updates to `interegular`.  Users can update `interegular` in conventional ways; there's no need to pin a version requirement here.

Reply:
I'm not sure if interegular is getting any more updates per https://github.com/MegaIng/interegular/pull/5#issuecomment-1877824290

I'll ask the author to push the update to pypi though.